### PR TITLE
feat: Add multimodal semantic convention for llm message contents

### PR DESCRIPTION
**Pull request by fjcasti1**

### CODE REVIEW

File: js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts [hunk 1]
```diff
@@ -204,6 +217,31 @@ export const MESSAGE_FUNCTION_CALL_ARGUMENTS_JSON =
  */
 export const MESSAGE_CONTENT =
   `${SemanticAttributePrefixes.message}.${MessageAttributePostfixes.content}` as const;
+/**
+ * The contents of the message sent to the LLM
```
Comment (mikeldking) [inline js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts:221]: Maybe highlight the plural aspect

File: js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts [hunk 2]
```diff
@@ -204,6 +217,31 @@ export const MESSAGE_FUNCTION_CALL_ARGUMENTS_JSON =
...
+ */
+export const MESSAGE_CONTENTS =
+  `${SemanticAttributePrefixes.message}.${MessageAttributePostfixes.contents}` as const;
+/**
+ * The contents of the message sent to the LLM
```
Comment (mikeldking) [inline js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts:226]: docstring is wrong

File: js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts [hunk 3]
```diff
@@ -204,6 +217,31 @@ export const MESSAGE_FUNCTION_CALL_ARGUMENTS_JSON =
...
+ */
+export const MESSAGE_CONTENT_TYPE =
+  `${SemanticAttributePrefixes.message_content}.${MessageContentsAttributePostfixes.type}` as const;
+/**
+ * The text content of the message sent to the LLM
```
Comment (mikeldking) [inline js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts:231]: copy pasta

File: js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts (hunk 2 above)
Comment (fjcasti1) [reply js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts:226]: Fixed

File: js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts (hunk 1 above)
Comment (fjcasti1) [reply js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts:221]: Done

File: js/.changeset/young-cobras-confess.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+"@arizeai/openinference-semantic-conventions": minor
+---
+
+Add multimodal semantic convention for llm message contents
```
Comment (mikeldking) [inline js/.changeset/young-cobras-confess.md:5]:
```suggestion
Add multimodal semantic conventions for llm message contents
```

**Review by mikeldking — APPROVED** [review]

**Pull request merged**